src/modules/client/components/clientAlerts/ClientAlert.tsx

```diff
@@ -0,0 +1,52 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import {
+  parseAndFormatDate,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import { ClientAlertFieldsFragment } from '@/types/gqlTypes';
+
+export type ClientAlertType = {
+  alert: ClientAlertFieldsFragment;
+  clientName: string;
+  showClientName?: boolean;
+};
+
+interface ClientAlertProps {
+  clientAlert: ClientAlertType;
+}
+const ClientAlert: React.FC<ClientAlertProps> = ({ clientAlert }) => {
+  const alert = clientAlert.alert;
+  const clientName = clientAlert.clientName;
+  const showClientName = clientAlert.showClientName;
+  const priority = alert.priority || 'low';
+
+  return (
+    <Alert icon={false} variant='withHeader' color={priority}>
+      <AlertTitle
+        sx={{
+          textTransform: 'capitalize',
+        }}
+      >
+        {priority} Priority Alert
+      </AlertTitle>
+      <Box className='MuiAlert-body'>
+        {showClientName && (
+          <Typography variant='body2' sx={{ pb: 1 }}>
+            {clientName}
+          </Typography>
+        )}
+        <Typography variant='body1' sx={{ pb: 1 }}>
+          {alert.note}
+        </Typography>
+        <Typography variant='body2' sx={{ fontSize: '12px' }}>
```

Comment:
Setting a font-size manually is a code smell - we should usually be able to match existing typography, or else something is off with the designs.

It was hidden away in figma, but I can tell this is a `caption` variant
<img width="251" alt="Screenshot 2024-02-02 at 9 30 19 AM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/2f1a5f41-af28-4a12-bf8c-baaab8419cf9">



```suggestion
        <Typography variant='caption'>
```